oap-server/server-library/library-server/src/main/java/org/apache/skywalking/oap/server/library/server/jetty/JettyServer.java

```diff
@@ -73,6 +73,12 @@ public void initialize() {
         LOGGER.info("http server root context path: {}", jettyServerConfig.getContextPath());
 
         server.setHandler(servletContextHandler);
+
+        JettyDefaultHandler defaultHandler = new JettyDefaultHandler();
+        ServletHolder defaultHolder = new ServletHolder();
+        defaultHolder.setServlet(defaultHandler);
+
+        servletContextHandler.addServlet(defaultHolder, defaultHandler.pathSpec());
```

Comment:
Can you elaborate more on this, isn't it the default behavior (returning `HttpServletResponse.SC_NOT_FOUND `) when there is no mapped servlet can be found? Why do we need a default handler to do this?

Reply:
In `Jetty`, if we don't set a default handler they will set one `Default404Servlet`
![image](https://user-images.githubusercontent.com/16773043/120281887-18424a00-c2ec-11eb-879f-25da555db2a9.png)
But this servlet doesn't override `Service`, so the `TRACE` and others methods could get in from here like a back door.